numpy/core/einsumfunc.py

```diff
@@ -884,39 +886,68 @@ def einsum(*operands, **kwargs):
     -----
     .. versionadded:: 1.6.0
 
+    The Einstein summation convention can be used to compute
+    many multi-dimensional array operations. Additionally, the explicit
+    mode of this function provides further flexibility that can be used
```

Comment:
It might be less confusing to clearly separate the explicit and non-explicit modes in the description

Reply:
Since this was a new observation in the docs I tried to thread it neatly into the existing material. I think you are right though and maybe highlighting different modes needs to be more prominent.